schemas/interface.yml

```diff
@@ -0,0 +1,31 @@
+- name: interface
+  title: Interface
+  group: 2
+  short: Fields to describe observer interface information.
+  description: >
+    The interface fields describe information about ingress and egress interfaces reported 
+    by an observer.
+
+  reusable:
+    top_level: false
+    expected:
+      - observer.ingress
+      - observer.egress
+
+  type: group
+  fields:
+
+    - name: id
+      level: core
+      type: keyword
+      example: 1
+      description: >
+        Interface ID as reported by the system (typically SNMP interface ID).
+
+    - name: name
+      level: extended
+      type: keyword
+      example: GigabitEthernet0/1, eth0, outside
```

Comment:
Please only put one example in `example:` sections. The rendering looks funny with multiple examples ([see here](http://ecs_752.docs-preview.app.elstc.co/guide/en/ecs/master/ecs-vlan.html)).

The paragraph in the description to lay out more examples, however, is perfect. (Until we add specific support for multiple examples)

Reply:
fixed